packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -352,5 +350,8 @@ export default class QuanticCaseClassification extends LightningElement {
   setFieldValue(value) {
     this.field.update(value);
     this._value = value;
+    if(this._errorMessage && value){
```

Comment:
Shouldn't you instead re-trigger the reportValidity? Does that make sense?

Reply:
Yeah! better 👌